frontend/src/metabase/querying/parameters/utils/query.ts

```diff
@@ -158,8 +158,8 @@ function getNumberParameterFilterClause(
   return match({ operator, values })
     .with(
       { operator: P.union("=", "!=") },
-      { operator: P.union(">=", "<="), values: [P.number] },
-      { operator: "between", values: [P.number, P.number] },
+      { operator: P.union(">=", "<="), values: [P._] },
+      { operator: "between", values: [P._, P._] },
```

Comment:
`P.bigint` exists. Would this be clearer if it was `P.union(P.number, P.bigint)`?

```suggestion
      { operator: P.union(">=", "<="), values: [P.union(P.number, P.bigint)] },
      { operator: "between", values: [P.union(P.number, P.bigint), P.union(P.number, P.bigint)] },
```

Reply:
Values are already correct. The intent is to check for array length only.